AutomatedTesting/Gem/PythonTests/Multiplayer/TestSuite_Main.py

```diff
@@ -13,19 +13,43 @@
 
 sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../automatedtesting_shared')
```

Comment:
nit: add a comment to why you are appending to the system path.

Reply:
removed! along with some imports i didn't need.